fmf.spec

```diff
@@ -1,16 +1,19 @@
 Name: fmf
-Version: 1.3.0
-Release: 1%{?dist}
+Version: 0.0.0
+Release: %autorelease
 
 Summary: Flexible Metadata Format
 License: GPLv2+
 BuildArch: noarch
 
-URL: https://github.com/psss/fmf
-Source0: https://github.com/psss/fmf/releases/download/%{version}/fmf-%{version}.tar.gz
+URL: https://github.com/teemtee/fmf
+Source0: https://github.com/teemtee/fmf/releases/download/%{version}/fmf-%{version}.tar.gz
```

Comment:
This will be pypi_source macro, no?

Reply:
Good point, we are fixing the release in this PR, so we can indeed use it